api/users/education-schema.json

```diff
@@ -0,0 +1,52 @@
+{
+	"type": "array",
+	"items": {
+		"type": "object",
+		"properties": {
+			"degree": {
+				"type": "string"
+			},
+			"startYear": {
+				"type": "integer",
+				"minimum": 1900
+			},
+			"startMonth": {
+				"type": "integer",
+				"minimum": 1,
+				"maximum": 12
+			},
+			"endMonth": {
+				"type": "integer",
+				"minimum": 1,
+				"maximum": 12
+			},
+			"endYear": {
+				"type": "integer",
+				"minimum": 1900
+			},
+			"ongoing": {
+				"type": "boolean"
+			},
+			"department": {
+				"type": "string"
+			},
+			"institution": {
+				"type": "string",
+				"minLength": 1
+			}
+		},
+		"required": [
+			"startYear",
+			"startMonth",
+			"ongoing",
+			"department",
+			"institution",
+			"degree"
+		],
+		"additionalProperties": false,
+		"dependencies": {
```

Comment:
No, they are not necessary because `startMonth` and `startYear` are both always required no matter what, so they're not really dependencies of each other.

Reply:
Hmm, that doesn't match current functionality. I'm able to add an education entry to my profile without a start date.

Reply:
Can you double check these dependencies with functionality on production? Start year and end year can be entered and saved without start month and end month, but months can't be saved without years. I also think there should be a `"endYear": "startYear"` dependency here because an end year can't be saved unless there's a start year.